.github/workflows/ci.yml

```diff
@@ -19,9 +19,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        Dtest: [ "A*,B*", "C*", "nonexistingtest" ]
+        Dtest: [ "FailA*", "FailB*", "PassA*" ]
     uses: ./.github/workflows/worker.yml
     with:
       script: ".github/scripts/run-maven-command.sh"
-      args: "test -Dtest='${{ matrix.Dtest }}' -Dsurefire.failIfNoSpecifiedTests=false -P skip-static-checks -Dweb.console.skip=true -Dmaven.javadoc.skip=true --fail-at-end"
+      args: "test -Dtest='${{ matrix.Dtest }}' -Dsurefire.failIfNoSpecifiedTests=false -P skip-static-checks -Dweb.console.skip=true -Dmaven.javadoc.skip=true -Dmaven.test.failure.ignore=true"
       jdk: ${{ inputs.jdk }}
+      upload_reports: true
+      artifact_prefix: "surefire-reports"
+      report_directory_pattern: "./**/surefire-reports/**/*.xml"
```

Comment:
don't try to limit/control at this granularity....its easier to say that you want or not

for now let's call it just artifact and by default it contains the test report xmls; with fine granularity at start later new artifact will be added instead of just reusing just one

Reply:
Sorry, could you please elaborate?

Do you mean removing `upload_reports`, `artifact_prefix`, and `report_directory_pattern` --- and passing only one input for the pattern (`artifact_pattern` or something)?

Reply:
Just one input; like artifact_prefix if its filled it will pack everything into that.

One job - one artifact, no need to have more.

Put all inclusions for that artifact.

In the end it will be the test xmls and some other stuff

Reply:
Have made some changes based on my understanding of the suggestion. Please let me know if it aligns with what you had in mind. Thanks!